dcor/independence.py

```diff
@@ -12,6 +12,15 @@
 import scipy.stats
 
 from ._dcor import u_distance_correlation_sqr
+
+## Additional modules for Multivariate dcov-based test of independence------------
+import math
+from ._dcor import u_distance_covariance_sqr, dist_sum, gamma_ratio, rndm_projection  
+from mpmath import*
```

Comment:
Do not do a wildcard import outside of a console session. It is bad practice (obscures the code and may cause difficult-to-debug errors).
`mpmath` is not a dependence of this package, and should not be necessary.

Reply:
I have commented this part